src/hooks/useProposal/useProposal.ts

```diff
@@ -158,15 +158,16 @@ export function useProposal(proposal: ProposalTypes.Struct) {
 
   const buildSupportedNamespaces = useCallback(
     (proposal: ProposalTypes.Struct) => {
-      const requiredNamespaces = proposal.requiredNamespaces;
+      const { requiredNamespaces, optionalNamespaces } = proposal;
+
       const supportedNamespaces: BuildApprovedNamespacesParams["supportedNamespaces"] =
         {};
 
-      if ("eip155" in requiredNamespaces) {
+      if ("eip155" in requiredNamespaces || "eip155" in optionalNamespaces) {
         supportedNamespaces[SupportedNamespace.EIP155] =
           buildEip155Namespace(requiredNamespaces);
       }
-      if ("mvx" in requiredNamespaces) {
+      if ("mvx" in requiredNamespaces || "mvx" in optionalNamespaces) {
         supportedNamespaces[SupportedNamespace.MVX] =
           buildMvxNamespace(requiredNamespaces);
       }
```

Comment:
you also need to pass `optionalNamespaces` 

but it goes nicely with a change I wanted to suggest before to directly pass the namespace and not the object with all namespaces to the build fonctions as shown below
`const eip155Namespace = requiredNamespaces["eip155"] || optionalNamespaces["eip155"]`